packages/database/package.json

```diff
@@ -15,6 +15,7 @@
     "dev": "supabase start && tsx scripts/createEnv.ts && supabase functions serve",
     "stop": "supabase stop",
     "check-types": "tsc --noEmit --skipLibCheck",
+    "check-schema": "npm run lint && supabase stop && npm run dbdiff",
```

Comment:
npm run lint does both the code and database linting. Maybe instead do `tsx scripts/lint.ts` here, so we only do the latter.
Otherwise lgtm.